### PR TITLE
Adjusted CNVKIT/BATCH for germline mode

### PR DESCRIPTION
**Pull request by SusiJo**
## PR checklist

This PR enables the `cnvkit/batch` module to run in `germline mode` in addition to `tumor_only` and `somatic mode`.

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/cnvkit/batch/main.nf [hunk 1]
```diff
@@ -62,28 +66,130 @@ process CNVKIT_BATCH {
...
-
-    cat <<-END_VERSIONS > versions.yml
-    "${task.process}":
-        cnvkit: \$(cnvkit.py version | sed -e "s/cnvkit v//g")
-    END_VERSIONS
-    """
+    // somatic_mode cram_input
+    if (tumor_cram && normal_cram){
+        """
+        samtools view -T $fasta $fai_reference $tumor -@ $task.cpus -o $tumor_out
```
Comment (FriederikeHanssen) [inline modules/cnvkit/batch/main.nf:72]: you don't need the funky ##idx## thing?

Comment (SusiJo) [reply modules/cnvkit/batch/main.nf:72]: not if the `fasta_fai` is passed along with the `fasta` reference genome

Comment (FriederikeHanssen) [reply modules/cnvkit/batch/main.nf:72]: ok I'll ask about this on slack

Comment (SusiJo) [reply modules/cnvkit/batch/main.nf:72]: if both files are not local and are accessed instead via an https://, s3:// or other URL then the  "##idx##" would be needed.

**Review by FriederikeHanssen — APPROVED** [review]

**Pull request merged**